README.md

```diff
@@ -23,11 +23,12 @@ setenv CMSSW_GIT_REFERENCE /cvmfs/cms.cern.ch/cmssw.git.daily
 git cms-init
 
 ```
- * Top branch needed to run updated double b tagger training on AK8 jets
+ * Top branch needed to run updated double b tagger training on AK8 jets, and latest Run II MET filters
 ```
 git fetch --tags btv-cmssw
 
 git cms-merge-topic -u cms-btv-pog:BoostedDoubleSVTaggerV3-WithWeightFiles-v1_from-CMSSW_8_0_8_patch1
```

Comment:
@dmajumder 
In the mean time Savvas sent a PR that got merged, so I need to do this addition on top of that, so this PR should not be merged any more (conflict), I close it.
To answer your question more specifically, I think we should be able to use the latest MET recommendations whatever they might be, even if they are temporary. I think the filters are going to be implemented eventually in the Rereco, but until then, we should use this temporary recipe from the JetMET POG.